<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_course</class>
 <widget class="QWidget" name="Create_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教务端-新增课堂</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>29</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>课程编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_number">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>29</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_credit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学分:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_term">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学期:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_clase">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_score">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>教室:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_create">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>建立</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_max_pelple">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>可选人数:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
